tests/functional/algos/test_algos.py

```diff
@@ -280,54 +281,39 @@ def test_with_evc(algorithm):
 )
 def test_parallel_workers(algorithm):
     """Test parallel execution with joblib"""
+    MAX_TRIALS = 30
     with OrionState() as cfg:  # Using PickledDB
 
         name = "{}_exp".format(list(algorithm.keys())[0])
 
-        base_exp = create_experiment(
-            name=name,
-            space=space_with_fidelity,
-            algorithms=algorithm_configs["random"],
-        )
-        base_exp.workon(rosenbrock, max_trials=10)
-
         exp = create_experiment(
             name=name,
             space=space_with_fidelity,
             algorithms=algorithm,
-            branching={"branch_from": name, "enable": True},
         )
 
-        assert exp.version == 2
-
-        exp.workon(rosenbrock, max_trials=30, n_workers=2)
+        exp.workon(rosenbrock, max_trials=MAX_TRIALS, n_workers=2)
 
         assert exp.configuration["algorithms"] == algorithm
 
-        trials = exp.fetch_trials(with_evc_tree=False)
-        assert len(trials) >= 20
-
-        trials_with_evc = exp.fetch_trials(with_evc_tree=True)
-        assert len(trials_with_evc) >= 30
-        assert len(trials_with_evc) - len(trials) == 10
+        trials = exp.fetch_trials()
+        assert len(trials) >= MAX_TRIALS
 
-        completed_trials = [
-            trial for trial in trials_with_evc if trial.status == "completed"
-        ]
-        assert 30 <= len(completed_trials) <= 30 + 2
+        completed_trials = [trial for trial in trials if trial.status == "completed"]
+        assert MAX_TRIALS <= len(completed_trials) <= MAX_TRIALS + 2
 
         results = [trial.objective.value for trial in completed_trials]
         best_trial = next(
             iter(sorted(completed_trials, key=lambda trial: trial.objective.value))
         )
 
         assert best_trial.objective.name == "objective"
-        assert abs(best_trial.objective.value - 23.4) < 1e-5
+        assert abs(best_trial.objective.value - 23.4) < 5.0
         assert len(best_trial.params) == 2
         fidelity = best_trial._params[0]
         assert fidelity.name == "noise"
         assert fidelity.type == "fidelity"
-        assert fidelity.value == 10
+        assert fidelity.value >= 1
```

Comment:
Can fidelity be < 1 ?

Reply:
Solved in https://github.com/Epistimio/orion/commit/634739d6b1ee76ffa8bdd63b310b90808a95acca